@core/cms/src/external-auth-integration/auth-providers/basic-auth/@types/index.ts

```diff
@@ -0,0 +1,61 @@
+export type BasicAuthString = `basic-${string}`;
+
+/**
+ * Represents the client credential for Basic HTTP Authentication.
```

Comment:
I found that in OAuth2, ICredentials may not a good name, it should be like IClientCredentials or sth

Reply:
is IClientCredentials meant to be shared with other types of auth, like Basic Auth?